### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by danrday**

### CODE REVIEW

File: modifyElements.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    var d = document.querySelector(`#${callback}`);
+    var idNum = callback.split("--")[1];
+    var color = carlot.getInventory()[idNum].color;
+    d.style.backgroundColor = "beige";
+    d.style.outline = `5px solid ${color}`;
+    carlot.uniformCards();
+  }
+ };
+
+ carlot.modifyCSS = function (callback) {
```
Comment (MAOstrander) [inline modifyElements.js]: One of the Technical Requirements was that this function accept two arguments: i) A car DOM element that was clicked on. ii) A color name.
This works (and is clever), but doesn't meet that specifications. :/

File: index.html [hunk 2]
```diff
@@ -4,12 +4,27 @@
...
 <body>
-  
 
+  <p class="smallScreen">Sorry this page currently only works at 1200px width and above.</p>
+
+  <nav>
+    <input type="text" id="textInput" class="form-control" placeholder="Edit Description" disabled>
+    <button type="submit" id="submitButton" class="btn btn-default" disabled>Submit</button>
+  </nav>
+
```
Comment (MAOstrander) [inline index.html:17]: I love how clean and simple this NavBar is, but it isn't the "standard Bootstrap navbar element" that was specified in the behavioral requirements.

File: activateElements.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+var Carlot = (function (carlot) {
+
+  var lastCardSelected; //set by clickHandler();
+  //defined these globally so that editDescript() updates DOM immediately
+  var selectedTargetNum;  //number corresponding to array value in JSON file, set by clickHandler();
+  var descriptionSelected; //set by clickHandler();
+  var textBox = document.getElementById("textInput");
+  var submitButton = document.getElementById("submitButton");
+
+  carlot.activateElements = function () {
```
Comment (MAOstrander) [inline activateElements.js]: Since I'm being overly particular about details, the requirement said to "Name the function activateEvents." ^.^

**Comment by danrday** [comment]
fixed the bugs

**Pull request merged**